CHANGELOG.md

```diff
@@ -16,7 +17,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - `ModelPipeline.register_pretrained_model` should persist the user-supplied
   estimator object indefinitely. (#331)
 - Fixed requirements.txt listing for `cloudpickle` -- `>=0.2`, not `<=0.2`. (#323)
-- Fixed issue in `civis.io.read_civis_sql` when returning data that contains 
+- Fixed issue in `civis.io.read_civis_sql` when returning data that contains
   double quotes. (#328)
```

Comment:
Perhaps we should add an entry of about fixing the version of pyyaml for Python 3.4